openmc/capi/cell.py

```diff
@@ -128,6 +132,19 @@ def fill(self, fill):
             indices = (c_int32*1)(-1)
             _dll.openmc_cell_set_fill(self._index, 1, 1, indices)
 
+    def get_temperature(self, instance=None):
```

Comment:
@pshriwise I'll change this to the two methods you mentioned in a bit, but is this what you had in mind by getters/setters with values of each cell instance?

Reply:
Pretty close. I was thinking that getting temperatures would still be like accessing a property of `openmc.capi.cell`, so something like:

```python
@property
def temperatures(self):
    temps = POINTER(c_double)()
    n_temps = c_int()
    _dll.openmc_get_cell_temperatures(self._index, temps, n_temps)
    temp_array = as_array(temps, (n.value,))
    return temp_array
```

`openmc.capi.material.densities` is a good reference for this.

This also makes me realize I messed up my proposed capi functions above :)

`openmc_get_cell_temperature` should probably be `openmc_get_cell_temperatures` with a signature like

`openmc_get_cell_temperatures(int32_t index, double **T, int* n)`

My bad!